techmatters/terraso-mobile-client
# feat: create all the color screens, minus the photo algorithm

## Description
Creates all the color screens, punting on a good number of things for now as the PR is already quite big. Needs to be merged after techmatters/terraso-client-shared#278 so we can update to the right client shared version on that repo's main branch.

### Checklist

### Related Issues
First step towards #129

### Verification steps
User should be able to navigate through all of the screens in the soil color design. Manual entry of soil color should work. See parent issue #129 for everything that is not yet working.
